iris/segmentation/__init__.py

```diff
@@ -109,7 +109,7 @@ def read_masks(image_id, user_id):
     user_mask = np.load(user_mask_file)
     return final_mask, user_mask
 
-def merge_masks(image_id):
+def merge_masks(image_id, npy=False):
     """Combine the masks of all users to a resulting mask"""
     final_mask_paths = get_mask_filenames(image_id, user_id="*")[0]
```

Comment:
The use of "*" here means that we get back a pattern like `segmentation/mountains/*_final.npy` which would match all user's final mask files, but now also matches `combined_final.npy` - which means if we run the merge twice, the first merged mask gets incorporated into the new one.